utils/validate_consistency.py

```diff
@@ -187,9 +187,11 @@ def validate_version():
     they match.
     """
     # Get version from python package
-    sys.path.insert(0, ROOT_DIR)
+    sys.path.insert(1, ROOT_DIR)
     import aiida  # pylint: disable=wrong-import-position
     version = aiida.__version__
+    # Clean up sys.path again
```

Comment:
This seems weird, but I guess this is the main reason why this PR is relevant, I presume? To make sure `sys.path` is properly cleaned up from AiiDA side?

Reply:
What is weird about this? It's just adding a path to import a module (which we need just for the aiida version) and then cleaning up the path again so that things aren't messed up further down the line.

Reply:
Feel free to suggest a better approach